packages/custom-field/test/visual/material/custom-field.test.js

```diff
@@ -2,7 +2,7 @@ import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { visualDiff } from '@web/test-runner-visual-regression';
 import '@vaadin/vaadin-template-renderer';
 import '@vaadin/vaadin-combo-box/theme/material/vaadin-combo-box.js';
-import '@vaadin/vaadin-date-picker/theme/material/vaadin-date-picker.js';
+import '@vaadin/date-picker/theme/material/vaadin-date-picker.js';
```

Comment:
Please also update `package.json` for the `custom-field` package.

Reply:
There was no dependency of `@vaadin/vaadin-date-picker` in the `package.json`. I'm assuming I must add it to the `devDependencies` list since it's only used in tests.

Reply:
Done.